microsoft/testsuites/network/netinterface.py

```diff
@@ -50,15 +54,33 @@ class NetInterface(TestSuite):
         priority=1,
         requirement=simple_requirement(network_interface=Synthetic()),
     )
-    def validate_netvsc_reload(self, node: Node) -> None:
+    def validate_netvsc_reload(self, node: Node, log_path: Path) -> None:
         self._validate_netvsc_built_in(node)
         network_interface_feature = node.features[NetworkInterface]
         # Test loading and unloading netvsc driver
         test_count = 0
         while test_count < self.NETVSC_RELOAD_TEST_COUNT:
             test_count += 1
             # Unload and load hv_netvsc
-            network_interface_feature.reload_module()
+            try:
+                network_interface_feature.reload_module()
+            except Exception as identifier:
+                # Sometimes the ssh session is inactive, call node.close() and retry
+                # can solve this problem. In this case, if there is no panic, then
+                # raise PassedException
+                node.log.debug(identifier)
+                serial_console = node.features[SerialConsole]
+                serial_console.check_panic(
+                    saved_path=log_path, stage="after_reload_netvsc", force_run=True
+                )
+                node.close()
+                remote_node = cast(RemoteNode, node)
+                try_connect(remote_node._connection_info)
+                node.log.debug("The node can be connected.")
+                raise PassedException(
```

Comment:
it is in a test loop, throw exception here, it may reduce test times?

Reply:
Agree, it needs to add another loop to retry here. If it's passed in retry, it should try next test_count, instead of raise PassedException.

Reply:
Done